packages/theme/css/palette/foreground-next.css

```diff
@@ -0,0 +1,22 @@
+.salt-theme.salt-theme-next[data-mode="light"] {
+  --salt-palette-foreground-active: var(--salt-color-blue-700);
+  --salt-palette-foreground-hover: var(--salt-color-blue-600);
+  --salt-palette-foreground-invert: var(--salt-color-white);
```

Comment:
the -invert is quite confusing to me without a default --salt-palette-foreground (what are we inverting?)